src/commands/pack.js

```diff
@@ -187,13 +187,13 @@ function getDevServerConfig() {
   });
 }
 
-function packProject() {
+function packProject(options) {
   return new Promise((resolve, reject) => {
     console.log(
       `${delimiter}: Running webpack...`
     );
     getWebpackConfig().then(
-      (config) => build(config).then(resolve, reject), reject
+      (config) => build(config, options.nobrowser).then(resolve, reject), reject
```

Comment:
can we pass the entire `options` object? I would like to be ready for new options that we may decide to add in the future.